crates/cxx-qt/src/lib.rs

```diff
@@ -50,3 +50,118 @@ pub trait CxxQtType {
 pub trait Locking {
     // empty
 }
+
+/// This trait can be implemented inside the [`#[cxx_qt::bridge]`](bridge) on any [CxxQtType] to define a
+/// custom constructor in C++ for the QObject.
+///
+/// The `Arguments` must be a tuple of CXX types that will be the arguments to the constructor in C++.
+///
+/// Note that implementing this trait will stop Cxx-Qt from generating a default constructor.
+///
+/// # Minimal Example
+/// ```no_run
+/// # #[macro_use] extern crate cxx;
```

Comment:
curious, why is that line required when we didn't need it in the doc tests in `cxx-qt-macro`?

Reply:
Hm, seems to work without this as well.
Might have been fixed by adding cxx as a dev-dependency, not sure.